templates/docs/db/fizz.md

```diff
@@ -26,7 +26,7 @@ create_table("todos") {
 }
 ```
 
-The `create_table` function will generate an `id` column of type `integer` that will auto-increment. This can be changed to use the [`UUID`](https://github.com/gobuffalo/uuid) type:
+The `id` column don't have to be an integer. For instance, your can use an [`UUID`](https://github.com/gobuffalo/uuid) type instead:
```

Comment:
Just some typos

```suggestion
The `id` column doesn't have to be an integer. For instance, you can use an [`UUID`](https://github.com/gobuffalo/uuid) type instead:
```

Im not sure if UUID needs `an` (als the pronunciation doesn't start with a vowel) but that doesn't really mater

Reply:
Thanks!